train.py

```diff
@@ -0,0 +1,300 @@
+import os
+import json
+import time
+import pickle
+import hashlib
+from typing import Callable, Iterable
+
+import hydra
+import torch
+import einops
+import torch.nn as nn
+from tqdm import tqdm
+from datasets import load_dataset
+from torch.utils.data import DataLoader
+from omegaconf import OmegaConf, DictConfig
+from torch.optim.lr_scheduler import LambdaLR
+
+import wandb
+from data.batch import Batch
+from model import make_model
+from utils import rate, distance
```

Comment:
```suggestion
from utils import learning_rate_schedule, euclidean_distance
```

I'm not sure about my version, but something more descriptive would be good :)